### PR TITLE
Manhwa latino

### PR DESCRIPTION
**Pull request by lberoiza**
Manhwa-Latino has changed the HTML Image-Attribute from src to data-src.

- The extension was updated
- The website has deactivate Cloudflare, it was deactivate from the extension to incremente the speed.

Best Regards, Tachiyomi-Team.

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: src/es/mahnwalatino/src/eu/kanade/tachiyomi/extension/es/manhwalatino/ManhwaLatino.kt [hunk 1]
```diff
@@ -40,10 +39,7 @@ class ManhwaLatino : ParsedHttpSource() {
     /**
      * Http Client with Cloudflare
      */
-    override val client: OkHttpClient = network.cloudflareClient.newBuilder()
-        .connectTimeout(10, TimeUnit.SECONDS)
-        .readTimeout(30, TimeUnit.SECONDS)
-        .build()
+    override val client: OkHttpClient = network.client.newBuilder().build()
```
Comment (alessandrojean) [inline src/es/mahnwalatino/src/eu/kanade/tachiyomi/extension/es/manhwalatino/ManhwaLatino.kt:43]: If you're not overriding the client you can safely remove this line IIRC.

Comment (lberoiza) [reply src/es/mahnwalatino/src/eu/kanade/tachiyomi/extension/es/manhwalatino/ManhwaLatino.kt]: it is done, I removed the line of code.

File: src/es/mahnwalatino/src/eu/kanade/tachiyomi/extension/es/manhwalatino/MLConstants.kt [hunk 2]
```diff
@@ -7,7 +7,8 @@ object MLConstants {
     const val searchMangaNextPageSelector = "link[rel=next]"
     const val latestUpdatesSelector = "div.slider__item"
     const val searchMangaSelector = "div.page-item-detail.manga"
-    const val popularMangaNextPageSelector = "nav.navigation-ajax"
+//    const val popularMangaNextPageSelector = "nav.navigation-ajax"
```
Comment (arkon) [inline src/es/mahnwalatino/src/eu/kanade/tachiyomi/extension/es/manhwalatino/MLConstants.kt:10]: Just remove it if you don't need it anymore.

File: src/es/mahnwalatino/src/eu/kanade/tachiyomi/extension/es/manhwalatino/ManhwaLatinoSiteParser.kt [hunk 3]
```diff
@@ -132,10 +136,13 @@ class ManhwaLatinoSiteParser(
      * Title, Address and thumbnail_url
      */
     fun getMangaFromList(element: Element): SManga {
+        print("getMangaFromList")
```
Comment (arkon) [inline src/es/mahnwalatino/src/eu/kanade/tachiyomi/extension/es/manhwalatino/ManhwaLatinoSiteParser.kt:139]: Remove this.

**Review by arkon — CHANGES_REQUESTED** [review]

File: src/es/mahnwalatino/src/eu/kanade/tachiyomi/extension/es/manhwalatino/MLConstants.kt (hunk 2 above)
Comment (lberoiza) [reply src/es/mahnwalatino/src/eu/kanade/tachiyomi/extension/es/manhwalatino/MLConstants.kt]: Ok, it was removed

File: src/es/mahnwalatino/src/eu/kanade/tachiyomi/extension/es/manhwalatino/ManhwaLatinoSiteParser.kt (hunk 3 above)
Comment (lberoiza) [reply src/es/mahnwalatino/src/eu/kanade/tachiyomi/extension/es/manhwalatino/ManhwaLatinoSiteParser.kt]: I didn't saw that, Thanks.
I had removed it

**Pull request merged**